nodejs/README.md

```diff
@@ -0,0 +1,148 @@
+# Pravega Nodejs Client
+
+This project provides a way to interact with [Pravega](https://cncf.pravega.io) via a Nodejs client.
+
+Pravega is an open source distributed storage service implementing Streams. It offers Stream as the main primitive for the foundation of reliable storage systems: a high-performance, durable, elastic, and unlimited append-only byte stream with strict ordering and consistency.
+
+## Install
+
+The client library can be installed using npm or yarn.
+
+```shell
+npm install @thekingofcity/pravega
```

Comment:
This should be generic

Reply:
I want to show that the current combination would result in a working package at my namespace. But turning it to something like `@pravega/pravega` requires far more effort on both npm registry and github actions. Things like using which email to register the npm and whether the binary url will work for the new address.